gapic/generator/generator.py

```diff
@@ -70,9 +71,14 @@ def get_response(self, api_schema: api.API) -> CodeGeneratorResponse:
         """
         output_files: Dict[str, CodeGeneratorResponse.File] = OrderedDict()
 
+        # TODO: handle sample_templates specially, generate samples.
+        sample_templates, other_templates = utils.partition(
+            self._env.loader.list_templates(),
+            lambda fname: os.path.basename(fname) == samplegen.TEMPLATE_NAME)
+
         # Iterate over each template and add the appropriate output files
         # based on that template.
-        for template_name in self._env.loader.list_templates():
+        for template_name in other_templates:
```

Comment:
Would it make sense to change `other_templates` to `client_templates` or `library_templates`?

Reply:
I like `client_templates`.